django-backend/fecfiler/transactions/views.py

```diff
@@ -509,6 +672,59 @@ def remove_transaction_from_report(self, request):
         transaction.reports.remove(report)
         return Response("Transaction removed from report")
 
+    def process_general_election_year_aggregates(self, transaction_instance):
```

Comment:
I'm wondering if we can name this function more descriptive - maybe update_previous_transactions_aggregates()?